### PR TITLE
Update push notification text

### PR DESCRIPTION
**Pull request by yuna878**
## Overview

Update push notification text

## Test Coverage

Manual testing

Design: 
image

Testing:
image

### CODE REVIEW

File: src/app/coursegrab/utils/push_notifications.py [hunk 1]
```diff
@@ -134,9 +133,7 @@ def send_emails(section, emails):
 
     message = Mail(
         from_email=("mailer@coursegrab.me", "CourseGrab"),
-        subject="Course ID {0}: {1}, {2} is now open!".format(
-            section["catalog_num"], section["title"], section["section"]
-        ),
+        subject=f"Course ID {section['catalog_num']}: {section['title']}, {section['section']} is now open!",
```
Comment (manishsaha) [inline src/app/coursegrab/utils/push_notifications.py:136]: Should we rewrite the subject to be consistent with the push notification message?

i.e. `subject=f"{serialized_section['subject_code']} {serialized_section['course_num']} {trimmed_section_name} is Now Open",`

**Review by manishsaha — APPROVED** [review]
Looks good, just minor comment on message consistency!

File: src/app/coursegrab/utils/push_notifications.py (hunk 1 above)
Comment (yuna878) [reply src/app/coursegrab/utils/push_notifications.py:136]: Yea I was also thinking about that - design hasn't iterated over what the email notifications would look like, but I guess I can make it consistent with push notifications for now!

**Review by alanna-zhou — APPROVED** [review]
yayyyy f stringssss

**Pull request merged**